packages/webpack-common/src/extractText.js

```diff
@@ -18,7 +18,7 @@ export {
  */
 function getLoaderConfigByType (context, webpackConfig, fileType) {
   const loaderConfig = webpackConfig.module.loaders.find(
-    (loader) => loader.test === context.fileType(fileType)
+    (loader) => regexEqual(loader.test, context.fileType(fileType))
```

Comment:
Is it really doing a deep clone of the regex (new regex with same value)? I wouldn't expect that... (Or: If it really works this way, maybe this is this the bug 😉)